internal/pkg/ecslogging/service.go

```diff
@@ -35,14 +37,26 @@ type ServiceClient struct {
 // WriteLogEventsOpts wraps the parameters to call WriteLogEvents.
 type WriteLogEventsOpts struct {
 	Follow    bool
-	Limit     int
+	Limit     *int64
 	StartTime *int64
 	EndTime   *int64
 	TaskIDs   []string
 	// OnEvents is a handler that's invoked when logs are retrieved from the service.
 	OnEvents func(w io.Writer, logs []HumanJSONStringer) error
 }
 
+func (o WriteLogEventsOpts) limit() *int64 {
+	if o.Limit != nil {
+		return o.Limit
+	}
+	if o.StartTime != nil || o.EndTime != nil {
```

Comment:
👍 Is there documentation of the default limits in our flag descriptions or usage help?

Reply:
great catch! since we now remove the default value for the flag, i think we should add to the description.